src/apps/material/material.tsx

```diff
@@ -22,9 +27,147 @@ const Material: React.FC<MaterialProps> = ({ pid }) => {
   }
 
   return (
-    <main>
+    <div className="material-page">
```

Comment:
I see that this is according to the design system. I general I think it is OK to use the `<main>` tag here for an app we assume will be the main content of a page.

Reply:
This makes sense. I corrected it :)